### DISCUSSION TITLE
Welcome to Week-0-Introduction Discussions!

### DISCUSSION THREAD

**Discussion by nmd2k** [Foundation-Learning-UETAILab/Week-0-Introduction#4] [Announcements]
## 👋 Welcome to the `UET-AILab`'s foundation machine learning course!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

 📢  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nmd2k** [comment]
👋 Chào mừng mọi người đến với Forum của Tuần 0, mọi người giới thiệu về bản thân ở đây nhé.

🙋‍♂️ Mình là Dũng, hiện đang là năm 4 ở UET. Mình đang học tập tại lab UET-AILab về MLOps, một chút về NLP.
Mình là người setup và điều hành cộng đồng trên github này, có vấn đề gì các bạn cứ `ping` mình nhé ✔️

> PS: Tiếng anh được khuyến khích, nhưng các bạn có thể sử dụng tiếng việt cho thoải mái nha ⛱️

**Comment by HaiNguyen2903** [comment]
👋 Chào mọi người, mình là Hải. Mình cũng là năm 4 cùng @manhdung20112000 , mọi người cần hỗ trợ gì cứ `ping` mình nhé 🙏

**Reply by nmd2k** [reply]
@HaiNguyen2903 ping

**Comment by nmd2k** [comment]
A warm welcome for @sang18112001 👋

**Reply by sang18112001** [reply]
em chào anh em mới tiếp cận môn này mong được các anh giúp đỡ ạ

**Reply by nmd2k** [reply]
Chào Sang nhé 👋, chúng mình ở đây để hỗ trợ bạn. Tuần này được xây dựng cũng cơ bản, chủ yếu để bạn làm quen với các thao tác của Github và cách sử dụng course này.

Cần giúp đỡ gì bạn cứ đề xuất nhé 🚀

**Reply by sang18112001** [reply]
dạ vâng ạ

**Comment by nmd2k** [comment]
Welcome @vduyhai to the course 🚀

**Comment by nmd2k** [comment]
Welcome @tungdop2 and @hoang1007 to the course 👋

**Comment by nmd2k** [comment]
Chào mừng @khanhvankhanh141 đến với khoá học 🚀

**Reply by khanhdv141** [reply]
Chào mọi người em mới vào lab mong được các anh chỉ dạy và giúp đỡ.

**Comment by vduyhai** [comment]
Rất vui khi được đồng hành với Lab 😀

Vào Th 4, 17 thg 11, 2021 vào lúc 16:40 Nguyễn Mạnh Dũng <
***@***.***> đã viết:

> Welcome @vduyhai <https://github.com/vduyhai> to the course 🚀
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Foundation-Learning-UETAILab/Week-0-Introduction/discussions/4#discussioncomment-1656792>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQCXF2YE6ZEWPOCHO6LW7TUMN2AFANCNFSM5HDZQOBQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Võ Duy Hải
 Military Technical Academy
P: (+84)795680415
Add: Hanoi, Vietnam
